kobweb-model/build.gradle.kts

```diff
@@ -0,0 +1,35 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    alias(libs.plugins.kotlin.jvm)
+}
+
+group = "com.varabyte.kobweb.intellij.model"
+version = libs.versions.kobweb.ide.plugin.get()
+
+repositories {
+    maven("https://www.jetbrains.com/intellij-repository/releases")
+    maven("https://cache-redirector.jetbrains.com/intellij-dependencies")
+}
+
+dependencies {
+    // Dependencies will ultimately be provided by the IDE.
+    compileOnly(gradleApi())
+    // The version doesn't matter too much here; just using something recent at the time of writing this build script.
+    // As long as the project compiles and the API is backwards compatible with latest IJ APIs, we should be good.
+    compileOnly("com.jetbrains.intellij.gradle:gradle-tooling-extension:233.13135.103")
```

Comment:
I intentionally didn't put this in the libs.versions.toml file because it feels like a one-off declaration. Maybe I'm overthinking it. I just didn't want this autocompleting for future modules we might create.

Reply:
I get your point. However we could decide to add the min and max intellij build number to our version file in general. This is not in the scope of this PR though so feel free to resolve this.

Reply:
Sounds good, thanks!

I'm not too sure what best practices for libs.versions.toml are. In general, I put things in there I think might be shared by multiple build scrips. Similar, I'm realizing, to how I decide whether to create a constant for a string value or not, sometimes. 

```kotlin
private const val PROJECT_NAME = "example"
app.createProject(PROJECT_NAME)

// vs

app.createProject("example")
```

Reply:
Okay then. Lets worry about this if we ever run into having to specify an intellij version again